internal/service/router.go

```diff
@@ -20,13 +20,16 @@ func Run(ctx context.Context, cfg config.Config) {
 			handlers.CtxLog(cfg.Log()),
 			handlers.CtxFormsQ(pg.NewForms(cfg.DB().Clone())),
 			handlers.CtxForms(cfg.Forms()),
+			handlers.CtxStorage(cfg.Storage()),
 		),
 	)
 	r.Route("/integrations/geo-forms-svc/v1", func(r chi.Router) {
 		r.Route("/form", func(r chi.Router) {
 			r.Use(handlers.AuthMiddleware(cfg.Auth(), cfg.Log()))
+			r.Get("/{id}", handlers.FormByID)
+			r.Get("/{nullifier}/last", handlers.FormByNullifier)
 			r.Post("/submit", handlers.SubmitForm)
-			r.Get("/{id}", handlers.GetForm)
+			r.Post("/lightweightsubmit", handlers.SubmitLightweightForm)
```

Comment:
while guys haven't integrated with it, rename to RESTful `/forms`. We will leave `/submit` as legacy, add comment in there that it's deprecated and slow.